### ISSUE TITLE
"Male" (fixed) Body Type Missing "Climb" Animations

### ISSUE THREAD

**Issue by jrconway3** [sanderfrenken/Universal-LPC-Spritesheet-Character-Generator#138]
Now that we've got expanded animation sets from https://github.com/sanderfrenken/Universal-LPC-Spritesheet-Character-Generator/issues/88, we can start working on plugging in the gaps.

"Climb" currently only has the "thin" body models from ElizaWy's repository. In addition, currently its using the smaller heads from ElizaWy, so I need to fix those up to get them configured correctly.

I already started working on this but I haven't finished it yet.

Once Male is done it'd be ideal for Muscular to get done shortly after.

"Teen", "Pregnant", and "Female" are all using the same Climb animations.

**Comment by jrconway3** [comment]
I have to keep in mind that if I do finish these assets, I should add myself as a credit to the "body" as well for adding these missing assets. But only for the male/muscular bodies as those are the ones I'm adding new assets to. I won't do it for simply compiling already existing assets together.

**Comment by jrconway3** [comment]
https://github.com/sanderfrenken/Universal-LPC-Spritesheet-Character-Generator/pull/145

This PR fixes the "Male" Climb animation, but not "Muscular" yet.

**Comment by jrconway3** [comment]
Added references to Emotes and the new Combat animations which have now been added.

**Comment by jrconway3** [comment]
So for the Combat animations, this is a weird one. But I think I am just going to drop Eliza's "Male" combat animations and stick with that for now. Because I noticed that if the body is twisted more, sometimes there is no differences between the body shapes. This is most notable in the "Shoot" animations beyond frame 2, where all the "male" bodies are the same size. So I'm just dropping the male bodies from Eliza's work as the thick male bodies for "combat" as well. We can change this later if its deemed necessary.

This doesn't work for Muscular as it needs to be distinct, but I'm not going out of my way to add the Muscular bodies right now. Maybe later I'll consider it.

**Comment by jrconway3** [comment]
I'm just going to shift "Muscular" over to #130 completely and remove this card.
